exercises/concept/high-score/.meta/exemplar.jl

```diff
@@ -0,0 +1,34 @@
+"""
+    add_player!(scores, name, score=0)
+
+Add a player to the score board.
+"""
+add_player!(scores, name, score=0) = push!(scores, name => score)
```

Comment:
No action required, but your earlier comment about `push!` confused me because I always do this as `scores[name] = score`. Didn't even know this was possible!

Reply:
I'd usually also do it as `scores[name] = score` but I think there's some benefit to introduce people to the idea that `push!` works on a large number of collections, even if in this particular case it may not be the most common use. We still show off `scores[name] = score` in the update method below.

Technically `push!(scores, name => score)` can even _update_ the value of `scores[name]` if `name` is already in `scores` but I find that use of `push!` slightly misleading.